concepts/variadic-functions/about.md

```diff
@@ -0,0 +1,57 @@
+# About
+Functions in general accept only a fixed number of arguments. A variadic function is a function that accepts a variable number of arguments. If the last parameter of a function definition is prefixed by ellipsis ..., then the function can accept any number of arguments for that parameter.
```

Comment:
We apply "one sentence per line" and code elements in the text should be written in backticks. See https://exercism.org/docs/building/markdown/markdown for reference.

I applied this here but this feedback also applies to some other places I think.

```suggestion
Functions in general accept only a fixed number of arguments.
A variadic function is a function that accepts a variable number of arguments.
If the last parameter of a function definition is prefixed by ellipsis `...`, then the function can accept any number of arguments for that parameter.
```

Reply:
got it will work on it

Reply:
resolved